panther_gazebo/launch/spawn.launch.py

```diff
@@ -0,0 +1,298 @@
+    z = LaunchConfiguration("z").perform(context)
+    roll = LaunchConfiguration("roll").perform(context)
+    pitch = LaunchConfiguration("pitch").perform(context)
+    yaw = LaunchConfiguration("yaw").perform(context)
+    publish_robot_state = LaunchConfiguration("publish_robot_state").perform(context)
+    namespace = LaunchConfiguration("namespace").perform(context)
+    add_map_transform = LaunchConfiguration("add_map_transform").perform(context)
+
+    robots_list = ParseMultiRobotPose("robots").value()
+    if len(robots_list) == 0:
+        robots_list = {
+            namespace: {"x": x, "y": y, "z": z, "roll": roll, "pitch": pitch, "yaw": yaw}
+        }
+
+    spawn_group = []
+    for idx, robot_name in enumerate(robots_list):
+        init_pose = robots_list[robot_name]
+        x, y, z, roll, pitch, yaw = [str(value) for value in init_pose.values()]
+
+        spawn_log = LogInfo(msg=[f"Launching namespace={robot_name} with init_pose={init_pose}"])
+
+        spawn_robot = Node(
+            package="ros_gz_sim",
+            executable="create",
+            arguments=[
+                "-name",
+                robot_name,
+                "-topic",
+                "robot_description",
+                "-x",
+                x,
+                "-y",
+                y,
+                "-z",
+                z,
+                "-Y",
+                yaw,
+            ],
+            namespace=robot_name,
+            output="screen",
+        )
+
+        gz_bridge = Node(
+            package="ros_gz_bridge",
+            executable="parameter_bridge",
+            name="gz_bridge",
+            parameters=[{"config_file": gz_bridge_config_path}],
+            namespace=robot_name,
+            output="screen",
+        )
+
+        bringup_launch = IncludeLaunchDescription(
+            PythonLaunchDescriptionSource(
+                PathJoinSubstitution(
+                    [
+                        FindPackageShare("panther_bringup"),
+                        "launch",
+                        "bringup.launch.py",
+                    ]
+                )
+            ),
+            launch_arguments={
+                "wheel_type": wheel_type,
+                "wheel_config_path": wheel_config_path,
+                "controller_config_path": controller_config_path,
+                "battery_config_path": battery_config_path,
+                "publish_robot_state": publish_robot_state,
+                "use_sim": "True",
+                "simulation_engine": "ignition-gazebo",
+                "namespace": robot_name,
+            }.items(),
+        )
+
+        ns_prefix = robot_name + "/" if robot_name else robot_name
+
+        map_transform = Node(
+            package="tf2_ros",
+            executable="static_transform_publisher",
+            name="static_tf_publisher",
+            namespace=robot_name,
+            output="screen",
+            arguments=[x, y, z, roll, pitch, yaw, "map", ns_prefix + "odom"],
```

Comment:
```suggestion
            arguments=[x, y, z, roll, pitch, yaw, "world", ns_prefix + "odom"],
```
I propose to change the frame name to world because there is no map :D